models/issue_comment.go

```diff
@@ -116,6 +116,32 @@ const (
 	CommentTagOwner
 )
 
+// EnableTag enable a specific tag on the CommentTag.
+func (ct CommentTag) EnableTag(tag CommentTag) CommentTag {
```

Comment:
How about change the name to `WithTag`?

Reply:
Sounds good to me.

Reply:
Ah, sorry. I mean the function name `EnableTag` -> `WithTag`, because I think it will return a new `CommentTag` which contains both the two `CommentTag`.

Reply:
Ah woops, sorry for the misunderstanding.